orangecanvas/canvas/items/graphicstextitem.py

```diff
@@ -113,10 +110,13 @@ def __updateDefaultTextColor(self):
         # type: () -> None
         if self.__styleState & QStyle.State_Selected \
                 and not self.__styleState & QStyle.State_Editing:
-            role = QPalette.HighlightedText
+            bgRole = QPalette.Light
```

Comment:
This modification was written before #177. Perhaps we should instead leave `role = QPalette.HighlightedText` from #177 in.